pootle/apps/pootle_app/management/commands/set_filetype.py

```diff
@@ -0,0 +1,68 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+import os
+
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pootle.settings'
+
+from django.core.management.base import CommandError
+
+from pootle_format.models import Format
+from pootle_project.models import Project
+
+from . import PootleCommand
+
+
+class Command(PootleCommand):
+    help = "Manage Store formats."
+
+    def add_arguments(self, parser):
+        super(Command, self).add_arguments(parser)
+        parser.add_argument(
+            'filetype',
+            action='store',
+            help="Format to set")
```

Comment:
It is a filetype, not a format. XLIFF format has two filetypes: `xlf` and `xliff`.